src/test/java/donnafin/logic/parser/AddressBookParserTest.java

```diff
@@ -1,30 +1,21 @@
 package donnafin.logic.parser;
 
-import static donnafin.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static donnafin.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
-import static donnafin.commons.core.Messages.MESSAGE_USE_HELP_COMMAND;
-import static donnafin.testutil.Assert.assertThrows;
-import static donnafin.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-import java.util.Arrays;
-import java.util.List;
-
-import org.junit.jupiter.api.Test;
-
-import donnafin.logic.commands.AddCommand;
-import donnafin.logic.commands.ClearCommand;
-import donnafin.logic.commands.DeleteCommand;
-import donnafin.logic.commands.ExitCommand;
-import donnafin.logic.commands.FindCommand;
-import donnafin.logic.commands.HelpCommand;
-import donnafin.logic.commands.ListCommand;
+import donnafin.logic.commands.*;
```

Comment:
I'm afraid this wouldn't pass check style right? And there are other files where you use wildcard imports too.

Reply:
My bad was working on my desktop , which did not allow me to change the import order statements. Have made a new commit that Appeases the checkstyle, and hopefully appeases you!